instance/static/html/instance/details.html

```diff
@@ -0,0 +1,137 @@
+<h3>
+  {{ instance.name }}
+</h3>
+<div class="panel">
+  <div class="row">
+    <div class="large-10 columns">
+      <ul class="instance-description">
+        <li>LMS: <a href="{{ instance.url }}" target="_new">{{ instance.url }}</a></li>
+        <li>Studio: <a href="{{ instance.studio_url }}" target="_new">{{ instance.studio_url }}</a></li>
+      </ul>
+    </div>
+    <div class="large-2 columns">
+      <!-- Placeholder area to put status images / buttons -->
+        <span class="status radius label"
+        ng-if="instance.active_appserver"
+        ng-class="{warning: !instance.active_appserver.is_healthy, success: instance.active_appserver.is_healthy}"
+        tooltip="{{ instance.active_appserver.status_description }}">
+    <i ng-attr-class="{{ instance.active_appserver.is_steady && (instance.active_appserver.is_healthy && 'fa fa-check' || 'fa fa-times') || 'fa fa-ellipsis-h' }}"></i>
+    {{instance.active_appserver.name}}: {{ instance.active_appserver.status_name }}
```

Comment:
@bradenmacdonald Nit: Add spaces around `instance.active_apperserver.name` (there are some more instances of this throughout the diff).